301-parameterized-linked-templates/azuredeploy.parameters.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "Virtual Network Name": {
+      "value": "GEN-UNIQUE-8"
+    },
+    "Operating System Image": {
+      "value": "Red Hat Enterprise Linux 7 (latest, LVM)"
```

Comment:
set defaultValues in the templates for any non-gen values where you can and then remove the params from here

Reply:
Fixed